app/views/transitive_dependencies/show.js.erb

```diff
@@ -0,0 +1,32 @@
+var all_deps = $('.deps_item').map(function() {
+	return $(this).attr('data-id');
+}).get();
+
+var gem_deps = <%=@deps_names.inspect.html_safe%>;
+
+$.each(gem_deps, function (idx,dep) {
+	if(all_deps.includes(dep)==true) {
```

Comment:
Showing duplicate dependency at the bottom looked nicer than this:

![Screenshot from 2019-07-19 10-42-17](https://user-images.githubusercontent.com/7680662/61510782-fe5a2f80-aa11-11e9-86fd-a4bd36c047c6.png)

Reply:
Is `==true` part really needed?